tests/migrations/test_executor.py

```diff
@@ -186,7 +187,13 @@ def fake_storer(phase, migration=None, fake=None):
                 (executor.loader.graph.nodes["migrations", "0001_initial"], False),
             ],
         )
-        executor.migrate([("migrations", "0001_initial")])
+        # Applying the migration should raise a database level error
```

Comment:
maybe add to this comment "...because we haven't given the --fake-initial option"